src/test/java/org/broadinstitute/hellbender/tools/IndexFeatureFileIntegrationTest.java

```diff
@@ -410,20 +416,77 @@ public void testVCFWithNoRecords() {
 
         final String[] args = {
-                "-I" ,  testFile.getAbsolutePath(),
+                "-I" ,  ENSEMBL_GTF_TEST_FILE.getAbsolutePath(),
                 "-O" ,  outName.getAbsolutePath()
         };
         final Object res = this.runCommandLine(args);
         Assert.assertEquals(res, outName.getAbsolutePath());
 
         final Index index = IndexFactory.loadIndex(res.toString());
         Assert.assertTrue(index instanceof LinearIndex);
-        Assert.assertEquals(index.getSequenceNames(), Arrays.asList("Chromosome"));
-        checkIndex(index, Arrays.asList("Chromosome"));
+        Assert.assertEquals(index.getSequenceNames(), Collections.singletonList("Chromosome"));
+        checkIndex(index, Collections.singletonList("Chromosome"));
+    }
+
+    @DataProvider
+    Object[][] provideForTestEnsemblGtfIndexQuery() {
+        return new Object[][] {
+                {
+                        new SimpleInterval("Chromosome", 3019160, 3020500),
+                        1,
+                        new SimpleInterval[] {new SimpleInterval("Chromosome", 3019161, 3020489)},
+                        new String[] {"b2879"},
+                        new String[] {"ssnA"},
+                },
+                {
+                        new SimpleInterval("Chromosome", 3286269, 3288786),
+                        4,
+                        new SimpleInterval[] {
+                                new SimpleInterval("Chromosome", 3285478, 3286269),
+                                new SimpleInterval("Chromosome", 3286270, 3287025),
+                                new SimpleInterval("Chromosome", 3287426, 3288010),
+                                new SimpleInterval("Chromosome", 3288090, 3288785),
+                        },
+                        new String[] {
+                                "b3140",
+                                "b3141",
+                                "b3142",
+                                "b3143",
+                        },
+                        new String[] {
+                                "agaD",
+                                "agaI",
+                                "yraH",
+                                "yraI",
+                        },
+                }
+        };
     }
 
+    @Test(dataProvider = "provideForTestEnsemblGtfIndexQuery")
+    public void testEnsemblGtfIndexQuery(final SimpleInterval interval,
+                                         final Integer expectedNumResults,
+                                         final SimpleInterval[] expectedFeatureIntervals,
+                                         final String[] expectedGeneIds,
+                                         final String[] expectedGeneNames) {
+        // Test that we can query the file:
+        final FeatureDataSource<GencodeGtfFeature> featureReader = new FeatureDataSource<>(ENSEMBL_GTF_TEST_FILE);
```

Comment:
This should be in a try() block so it gets closed.

Reply:
Ah!  Yup.  Fixed.